Geometry/CMSCommonData/python/cmsExtendedGeometry2023devXML_cfi.py

```diff
@@ -22,37 +22,37 @@
         'Geometry/CMSCommonData/data/cavern.xml',
         'Geometry/TrackerCommonData/data/PhaseII/trackerParameters.xml',
         'Geometry/TrackerCommonData/data/pixfwdCommon.xml',
-	'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdMaterials.xml',
-	'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdCylinder.xml', 
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwd.xml', 
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdDisks.xml', 
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdInnerDisk1.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdInnerDisk2.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdInnerDisk3.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdInnerDisk4.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdInnerDisk5.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdInnerDisk6.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdInnerDisk7.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdOuterDisk1.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdOuterDisk2.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdOuterDisk3.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdOuterDisk4.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdOuterDisk5.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdOuterDisk6.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdOuterDisk7.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdOuterDisk8.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdOuterDisk9.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdOuterDisk10.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdblade1.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdblade2.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdblade3.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdblade4.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdblade5.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdblade6.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdblade7.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdblade8.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdblade9.xml',
-        'Geometry/TrackerCommonData/data/PhaseII/TiltedTracker/pixfwdblade10.xml',
+	'Geometry/TrackerCommonData/data/PhaseII/FlatTracker/pixfwdMaterials.xml',
```

Comment:
@lgray and @boudoul - I think, an agreement was 2023sim is flat tracker and 2023dev is tilted tracker. Perhaps, it would be better to rename scenarios (2023Flat and 2023Tilted) to avoid such confusion in the future?